src/main/java/nl/knaw/dans/bridge/plugin/lib/common/IResponseData.java

```diff
@@ -0,0 +1,28 @@
+package nl.knaw.dans.bridge.plugin.lib.common;
+
+import java.util.Optional;
+
+/*
+ * Created by Eko Indarto
+ */
+public interface IResponseData {
+
+    String getResponse();
+
+    default Optional<String> getState()  {
+        return Optional.empty();
+    }
+
+    default Optional<String> getPid() {
```

Comment:
PID, format depends on type of PID... DOI in case of EASY

Reply:
http://www.ncdd.nl/en/pid/
"A Persistent Identifier (PID) is a unique identification code attached to a digital object and registered at an agreed location."
"There are multiple PID systems. But which system is most suited to your situation: Archival Resource Keys (ARKs), Digital Object Identifiers (DOIs), Handle, OpenURL, Persistent Uniform Resource Locators (PURLs) or Uniform Resource Names (URNs"

Reply:
We don't know anything about the pID, what kind of type is is, but we prefer the resolvable persistent url, I think. This is just documentation no implementation.